kalliope/core/RestAPI/FlaskAPI.py

```diff
@@ -22,11 +25,20 @@ def __init__(self, app, port=5000, brain=None):
         self.app = app
         self.port = port
         self.brain = brain
+        self.allow_cors_request = allow_cors_request
 
         # Flask configuration remove default Flask behaviour to encode to ASCII
         self.app.url_map.strict_slashes = False
         self.app.config['JSON_AS_ASCII'] = False
 
+        if self.allow_cors_request is True:
+            try:
+              from flask_cors import CORS, cross_origin
```

Comment:
Yes, but I think it's not a clean way to ask the user to add a lib after install. I think it's not a big deal to add it by default to keep a simple installation.
Furthermore, the CORS is mandatory to allow JS framework to work. And so I also think that it should be activated and hard coded to be active.

About the PR itself, I'll add a comment in the main thread.